modules/storage-rocksdb/src/test/java/org/apache/ignite/internal/storage/rocksdb/RocksDbMvPartitionStorageTest.java

```diff
@@ -51,15 +50,13 @@ public class RocksDbMvPartitionStorageTest extends AbstractMvPartitionStorageTes
     @BeforeEach
     void setUp(
             @WorkDirectory Path workDir,
-            @InjectConfiguration("mock.flushDelayMillis = 0")
```

Comment:
Is this setting configured for this test somehow now?

Reply:
Everything works without it, and I feel like it is irrelevant in most tests